python-iml-manager.spec

```diff
@@ -294,6 +294,11 @@ fi
 mkdir -p %{_localstatedir}/lib/rpm-state/%{name}/
 cp -r /var/lib/chroma/* %{_localstatedir}/lib/rpm-state/%{name}/
 
+%posttrans -n python2-%{pypi_name}-libs
+if [ -d %{python_sitelib}/iml_manager-5.0.*.egg-info ]; then
```

Comment:
This will cause issues if the glob matches more than one directory.
```
nclark@ieel-storage ~/integrated-manager-lustre/chroma_core>$ ls
app.py  app.pyc  fixtures/  __init__.py  __init__.pyc  lib/  management/  migrations/  models/  plugins/  services/  south_migrations/
nclark@ieel-storage ~/integrated-manager-lustre/chroma_core>$ [ -f *.pyc ] && echo delete
-bash: [: app.pyc: binary operator expected
```

Reply:
I would do:
```
if ls -1 %{python_sitelib}/|grep -q 'iml_manager-5.0.*.egg-info'; then
```